src/main/java/hudson/plugins/throttleconcurrents/ThrottleQueueTaskDispatcher.java

```diff
@@ -682,5 +689,26 @@ private int getMaxConcurrentPerNodeBasedOnMatchingLabels(
         return maxConcurrentPerNodeLabeledIfMatch;
     }
 
+    private void updatePauseAction(Task task, CauseOfBlockage cause) {
+        if (task instanceof PlaceholderTask) {
+            PlaceholderTask placeholderTask = (PlaceholderTask)task;
+            try {
+                FlowNode flowNode = placeholderTask.getNode();
```

Comment:
`ExecutorStepExecution.PlaceholderTask#getNode` can return `null`, but you're not doing a `null` check on `flowNode`.

Reply:
Done